napalm/junos.py

```diff
@@ -100,7 +103,7 @@ def get_facts(self):
             'os_version': output['version'],
             'hostname': output['hostname'],
             'fqdn': output['fqdn'],
-            'uptime': uptime,
+            'uptime': string_parsers.convert_uptime_string_seconds(uptime),
```

Comment:
Could you on top of that change the initialization of uptime: uptime = None to uptime = 0 ? Calling convert_uptime_string_seconds trips over uptime being None